units/heroes/Running_Heroes.cfg

```diff
@@ -0,0 +1,203 @@
+#textdomain wesnoth-Danse_Macabre
+[unit_type]
+    id=Running Hero
+    [base_unit]
+        id=Running Corpse
+    [/base_unit]
+    advances_to=Brainless Hero
+
+    # should be enabled in the end, since we
+    # a) don't want players in the beginning to see what heros will appear later
+    # b) nor do they have to see how the heroes look on higher levels (keeping some surprise)
+    # c) like the base_unit, they are shown in the help as advancements of the Soulless, which looks a bit odd
```

Comment:
Well, we inherit from Running Corpse here which has this:

> [advancefrom]
        unit=Soulless
        experience=35
[/advancefrom]

You mean Pech, right :) ? Drake and gryphon variations do have same graphics but there is a difference in movement costs and defense between them. The gryphon variation is better for her and she was like that with her own unit types.